sql/006_holdings.sql

```diff
@@ -11,7 +11,7 @@ CREATE TABLE `holdings` (
   `n_enum_chron` varchar(255) NULL,
   `status`       enum('CH', 'LM', 'WD') NULL,
   `condition`    enum('BRT', '') NULL,
-  `gov_doc_flag` tinyint(1) DEFAULT '0' NOT NULL,
+  `gov_doc_flag` tinyint(1),
```

Comment:
The mongo implementation allows the gov doc flag to be nil, and it doesn't seem worth quibbling over the behavior here -- especially given that we aren't using this flag for anything AFAIK.

Reply:
Yes, you are correct, we need to allow it to be unspecified.
Worth repeating what I said in our Zoom earlier today:
Most holdings don't report govdocness, and it would be incorrect to apply default=0, since a nil value is really more of a "n/a" or a "don't know/care".